bin/configurator/static/_includes/en/configurator/tab/ci/github-actions/simple/kubernetes-runner/linux/buildah/infra.md.liquid

```diff
@@ -134,9 +135,11 @@ EOF
 > **Note**
 > For greater security, consider creating a more restricted ClusterRole/Role and using it instead of the `cluster-admin` cluster role above.
 
-### Install the runner set
+#### Deploy
 
```

Comment:
Just move values.yaml right here, no extra sections needed. Return headings to `###`.